plugins/modules/grafana_dashboard.py

```diff
@@ -12,99 +12,110 @@
     'metadata_version': '1.1'
 }
 
-DOCUMENTATION = '''module: grafana_dashboard
+DOCUMENTATION = '''
+---
+module: grafana_dashboard
 author:
-- Thierry Sallé (@seuf)
+  - Thierry Sallé (@seuf)
+version_added: "2.5"
```

Comment:
I think @gundalow removed the ‘version_added’ because it does not make sense anymore with collection

Reply:
`ansible-test` will require `version_added`
As Collections have their own release cycle we will need to decide on a new versioning number scheme. Once that's decided on I'll do a bulk update of all the files in the Collections.

For the moment, new modules and arguments should have `version_added: '2.10'`.